components/core/src/streaming_archive/writer/File.hpp

```diff
@@ -58,7 +58,11 @@ namespace streaming_archive { namespace writer {
                 m_segmentation_state(SegmentationState_NotInSegment),
                 m_is_metadata_clean(false),
                 m_is_written_out(false),
-                m_is_open(false)
+                m_is_open(false),
+                m_timestamps(nullptr),
```

Comment:
Why the explicit initialization to null?

Reply:
I am not familiar with unique ptr. If I don't null initialize them, will they just be default initialized?

Reply:
Yep, they start out empty.